src/Oscoin/Node.hs

```diff
@@ -199,13 +219,17 @@ getPathLatest
     -> NodeT c tx st s i m (Maybe (StateHash c, QueryVal st))
 getPathLatest path = do
     bs <- asks hBlockStore
-    stateHash <- blockStateHash . blockHeader <$> BlockStore.getTip (hoistBlockStore liftIO bs)
+    stateHash <- blockStateHash . blockHeader <$> BlockStore.getTip (hoistBlockStoreReader liftIO bs)
     result <- getPath stateHash path
     for result $ \v ->
         pure (stateHash, v)
 
-getBlocks :: (MonadIO m) => Depth -> NodeT c tx st s i m [Block c tx (Sealed c s)]
-getBlocks d = withBlockStore (`BlockStore.getBlocks` d)
+getBlocks
+    :: MonadIO m
+    => Depth
+    -> NodeT c tx st s i m [Block c tx (Sealed c s)]
+getBlocks d =
+  Chrono.toNewestFirst <$> withBlockStore (`BlockStore.getBlocks` ByDepth d)
```

Comment:
I haven't gotten to the definition of `Chrono.toNewestFirst` yet, but this reads funny: why is a time-related function operating on blocks? I guess I'd expect `Blockchain.toNewestFirst` if anything.